src/modules/launcher/Wox.Infrastructure/Wox.Infrastructure.csproj

```diff
@@ -46,5 +46,6 @@
     <PackageReference Include="System.Drawing.Common" Version="6.0.0" />
     <PackageReference Include="System.IO.Abstractions" Version="12.2.5" />
     <PackageReference Include="System.Runtime" Version="4.3.1" />
+    <PackageReference Include="TinyPinyin.Net" Version="1.0.2" />
```

Comment:
this is interesting, there are two variants here.  https://www.nuget.org/packages/TinyPinyin/ and https://www.nuget.org/packages/TinyPinyin.Net.  The version linked here does have a far higher usage count